internal/cli/cmd/sysdump.go

```diff
@@ -74,6 +74,9 @@ func initSysdumpFlags(cmd *cobra.Command, options *sysdump.Options, optionPrefix
 	cmd.Flags().StringVar(&options.CiliumEnvoyLabelSelector,
 		optionPrefix+"cilium-envoy-label-selector", sysdump.DefaultCiliumEnvoyLabelSelector,
 		"The labels used to target Cilium Envoy pods")
+	cmd.Flags().StringVar(&options.CiliumHelmReleaseName,
+		"cilium-helm-release-name", sysdump.DefaultCiliumHelmReleaseName,
+		"Cilium Helm release name to get the values from the sysdump")
```

Comment:
"The Cilium Helm release name for which to get values"

(referring to the sysdump here is redundant)